controllers/members.js

```diff
@@ -33,8 +33,8 @@ const getIdleMembers = async (req, res) => {
   try {
     const allMembers = await fetchMembers()
     const taskParticipants = await tasks.fetchActiveTaskMembers()
-    const idleMembers = allMembers?.filter(({ id }) => !taskParticipants.has(id))
-    const idleMemberUserNames = idleMembers?.map((member) => member.username)
+    const idleMembers = allMembers.filter(({ id }) => !taskParticipants.has(id))
+    const idleMemberUserNames = idleMembers.map((member) => member.username)
```

Comment:
Why was this removed? We need those checks in place
It's not by mistake, it's called optional chaining: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

You maybe having older version of Node, that's why it didn't work in your machine, try updating node in your machine

Reply:
ok. Did not know about that. 👍